source/extensions/filters/http/jwt_authn/jwks_cache.cc

```diff
@@ -63,6 +66,17 @@ class JwksDataImpl : public JwksCache::JwksData, public Logger::Loggable<Logger:
     return setKey(std::move(jwks), getRemoteJwksExpirationTime());
   }
 
+  TokenCache& getTokenCache() override {
+    if (token_cache_ == nullptr) {
+      if (jwt_provider_.token_cache_size() > 0) {
```

Comment:
we should check "enable_jwt_cache" config.  If false, not to create token_cache.

Reply:
I've make sure that will only call if `enable_token_cache` is true.